### PR TITLE
[TIMOB-24912] Android: Running a module after it was built fails on Windows.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24912

Change parameters to work both on macOS and Windows.

**Description:**
On Windows Node child processes need to be run in a new console. This changes the command to point to the node executable and adds the old command as a parameter. This way it should work on both operating systems.

### CODE REVIEW

**Review by cb1kenobi — APPROVED** [review]
CR'd and FR'd. **APPROVED**

File: android/cli/commands/_buildModule.js [hunk 1]
```diff
@@ -1803,12 +1803,13 @@ AndroidModuleBuilder.prototype.runModule = function (next) {
...
 
-	function runTiCommand(cmd, args, logger, callback) {
-		// when calling a Windows batch file, we need to escape ampersands in the command
-		if (process.platform == 'win32' && /\.bat$/.test(cmd)) {
-			args.unshift('/S', '/C', cmd.replace(/\&/g, '^&'));
-			cmd = 'cmd.exe';
-		}
+	function runTiCommand(cmd, args, logger, callback) {		
+
+        // when calling on Windows, we need to escape ampersands in the command
```
Comment (cb1kenobi) [inline android/cli/commands/_buildModule.js]: Please fix the indention here. We use tabs, not spaces.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]
Needs some coding style fixes.

**Comment by ypbnv** [comment]
@cb1kenobi Updated with proper indentation.

File: android/cli/commands/_buildModule.js [hunk 2]
```diff
@@ -1803,11 +1803,11 @@ AndroidModuleBuilder.prototype.runModule = function (next) {
 		}
 	}
 
-	function runTiCommand(cmd, args, logger, callback) {
-		// when calling a Windows batch file, we need to escape ampersands in the command
-		if (process.platform == 'win32' && /\.bat$/.test(cmd)) {
-			args.unshift('/S', '/C', cmd.replace(/\&/g, '^&'));
-			cmd = 'cmd.exe';
+	function runTiCommand(cmd, args, logger, callback) {
```
Comment (cb1kenobi) [inline android/cli/commands/_buildModule.js]: Sorry to knit pick, but you have trailing whitespace.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

File: android/cli/commands/_buildModule.js (hunk 2 above)
Comment (ypbnv) [inline android/cli/commands/_buildModule.js]: I have totally missed that.

**Comment by ypbnv** [comment]
@cb1kenobi Updated.

**Review by cb1kenobi — APPROVED** [review]
Good to go. **APPROVED**!

**Pull request merged**